vmm/src/device_manager.rs

```diff
@@ -3475,6 +3535,17 @@ impl DeviceManager {
             .notify(AcpiNotificationFlags::POWER_BUTTON_CHANGED)
             .map_err(DeviceManagerError::PowerButtonNotification)
     }
+
+    #[cfg(target_arch = "aarch64")]
+    pub fn notify_power_button(&self) -> DeviceManagerResult<()> {
```

Comment:
Adding more information about why the power-button is not restricted to ACPI on Arm:
- http://lists.infradead.org/pipermail/linux-arm-kernel/2015-April/340036.html
- https://linux-arm-kernel.infradead.narkive.com/0BD9XhYo/discussion-how-to-implement-external-power-down-for-arm